app-antivirus/clamav-gui/patches/clamav_gui-1.0.9.patchset

```diff
@@ -0,0 +1,100 @@
+From e5a35ae42409cf7304b2ba2f0476ba8a45e185ec Mon Sep 17 00:00:00 2001
+From: Begasus <begasus@gmail.com>
+Date: Sat, 19 Oct 2024 12:55:41 +0200
+Subject: Fix installation paths
+
+
+diff --git a/clamav-gui.pro b/clamav-gui.pro
+old mode 100755
+new mode 100644
+index 2aee353..c53b347
+--- a/clamav-gui.pro
++++ b/clamav-gui.pro
+@@ -132,64 +132,64 @@ INSTALLS += target \
+     manpages_pt \
+     documents
+ 
+-target.path = /$(DESTDIR)/usr/bin
++target.path = /$(DESTDIR)/bin
+ target.files = clamav-gui
+ 
+-icon16.path = /$(DESTDIR)/usr/share/icons/hicolor/16x16/apps
++icon16.path = /$(DESTDIR)/data/icons/hicolor/16x16/apps
+ icon16.files = extra/icon16/clamav-gui.png
+ 
+-icon22.path = /$(DESTDIR)/usr/share/icons/hicolor/22x22/apps
++icon22.path = /$(DESTDIR)/data/icons/hicolor/22x22/apps
+ icon22.files = extra/icon22/clamav-gui.png
+ 
+-icon32.path = /$(DESTDIR)/usr/share/icons/hicolor/32x32/apps
++icon32.path = /$(DESTDIR)/data/icons/hicolor/32x32/apps
+ icon32.files = extra/icon32/clamav-gui.png
+ 
+-icon48.path = /$(DESTDIR)/usr/share/icons/hicolor/48x48/apps
++icon48.path = /$(DESTDIR)/data/icons/hicolor/48x48/apps
+ icon48.files = extra/icon48/clamav-gui.png
+ 
+-icon64.path = /$(DESTDIR)/usr/share/icons/hicolor/64x64/apps
++icon64.path = /$(DESTDIR)/data/icons/hicolor/64x64/apps
+ icon64.files = extra/icon64/clamav-gui.png
+ 
+-icon128.path = /$(DESTDIR)/usr/share/icons/hicolor/128x128/apps
++icon128.path = /$(DESTDIR)/data/icons/hicolor/128x128/apps
+ icon128.files = extra/icon128/clamav-gui.png
+ 
+-icon256.path = /$(DESTDIR)/usr/share/icons/hicolor/256x256/apps
++icon256.path = /$(DESTDIR)/data/icons/hicolor/256x256/apps
+ icon256.files = extra/icon256/clamav-gui.png
+ 
+-desktopfile.path = /$(DESTDIR)/usr/share/applications
++desktopfile.path = /$(DESTDIR)/data/applications
+ desktopfile.files = extra/ClamAV-GUI.desktop
+ 
+-servicemenufile.path = /$(DESTDIR)/usr/share/kio/servicemenus
++servicemenufile.path = /$(DESTDIR)/data/kio/servicemenus
+ servicemenufile.files = extra/ClamAV-GUI-ServiceMenu.desktop
```

Comment:
Do these even need to be patched? They are removed in the recipe anyway. Couldn't you just `rm -rf $prefix/usr` instead?

Reply:
fixed :)